tutorials/buoyantSimpleFoam-aster/precice-config.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<precice-configuration>
+  <log>
+    <sink enabled="true" output="stdout" type="stream" filter= "(%Severity% >= trace)"/>
+	<sink type="file" output="debug.log" filter="(%Severity% >= trace)" enabled="true" />
```

Comment:
At the end, do we need this additional sink? If yes, add a comment.

Reply:
No we didn't need the additional sink. I have removed it.